run_devstack_integration_tests.sh

```diff
@@ -40,14 +40,33 @@ pip install -r test_requirements.txt
 # output the packages which are installed for logging
 pip freeze
 
-# adjust test files for integration tests
-cp /edx/app/edxapp/edx-platform/setup.cfg .
+mkdir -p test_root  # for edx
 
-# Increase max line length
-sed -i 's/\[tool:pytest\]/\[tool:pytest\]\npep8maxlinelength = 119\n/' ./setup.cfg
+set +e
 
-mkdir -p test_root  # for edx
+pytest tests --cov .
```

Comment:
1. Wouldn't hurt to include a short comment above these commands explaining why we're not running pep8/pylint via pytest plugins
1. Just wondering: what do the `coverage xml` and `codecov` do (last 2 lines)? Is it necessary to run `pytest` with `--cov` in addition to those commands?

Reply:
The coverage data is produced as a `.coverage` file which is raw binary data. `coverage xml` converts that file to an xml file that `codecov` can read, and `codecov` will upload that data to codecov.io